### ISSUE TITLE
Can this run on TiviMate

### ISSUE THREAD

**Issue by BuddyChewChew** [gogetta69/public-files#1]
I keep getting: An error occurred: HttpDataSource$HttpDataSourceException.  trying to load.
 https://github.com/gogetta69/public-files/blob/main/playlist.m3u8

**Comment by gogetta69** [comment]
You can't just load this playlist directly. This is a playlist that is used along with

https://github.com/gogetta69/TMDB-To-VOD-Playlist

**Comment by BuddyChewChew** [comment]
I was wondering about the VOD playlist. 
https://github.com/gogetta69/public-files/blob/main/playlist.m3u8

It loads but links error out in TiviMate.

**Comment by gogetta69** [comment]
Yes, but if you look in the file you will see the [[SERVER_URL]] placeholder. This is replaced by the servers ip address. Install the script and it will work.
